pkgs/top-level/aliases.nix

```diff
@@ -600,6 +600,7 @@ mapAliases ({
   liblrdf = throw "'liblrdf' has been renamed to/replaced by 'lrdf'"; # Converted to throw 2022-02-22
   libmsgpack = throw "'libmsgpack' has been renamed to/replaced by 'msgpack'"; # Converted to throw 2022-02-22
   libosmpbf = throw "libosmpbf was removed because it is no longer required by osrm-backend";
+  libpng_apng = libpng; # Added 2021-03-21
```

Comment:
Please add a warnings as old name being deprecated.

Reply:
Why? This should already prevent people from adding it to nixpkgs, since ofborg evaluates with `allowAliases = false` iirc.

I can turn it into a `throw`, but I'm not sure that's better.

Reply:
I think with the scripted approach to alias management we have now we can have them phased out over time, no?

Reply:
The throw helps people to move expressions to the new one and over time avoids legacy code.